Content.Client/UserInterface/Systems/Actions/Controls/ActionPageButtons.xaml

```diff
@@ -13,7 +13,7 @@
         <Control HorizontalExpand="True" SizeFlagsStretchRatio="2"/>
         <Label Text="1" SizeFlagsStretchRatio="1" Name="Label" Access="Public" />
         <Control HorizontalExpand="True" SizeFlagsStretchRatio="2"/>
-        <TextureButton TexturePath="/Textures/Interface/Nano/right_arrow.svg.192dpi.png"
```

Comment:
The icons should probably just be moved to the default theme instead using a relative path like this. I.e., just move them into  `Interface\Default`.